hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -396,7 +407,14 @@ public AbfsRestOperation renameIdempotencyCheckOp(
   }
 
   public AbfsRestOperation append(final String path, final long position, final byte[] buffer, final int offset,
```

Comment:
append without tracingContext as a method should be removed ?

Reply:
yes, removed